pxr/base/arch/align.cpp

```diff
@@ -55,7 +55,7 @@ ArchAlignedAlloc(size_t alignment, size_t size)
 #elif defined(ARCH_OS_WINDOWS)
     return _aligned_malloc(size, alignment);
 #else
-    return std::aligned_alloc(alignment, size);
+    return aligned_alloc(alignment, size);
```

Comment:
I think we'll want to change the corresponding to call to std::free in ArchAlignedFree to regular free for consistency.

Reply:
Can do - but just noticed that would make the first `#if` branch and the `#else` branch identical:

```cpp
#if defined(ARCH_OS_DARWIN) || (defined(ARCH_OS_LINUX) && defined(__GLIBCXX__) && !defined(_GLIBCXX_HAVE_ALIGNED_ALLOC))
    free(ptr);
#elif defined(ARCH_OS_WINDOWS)
    _aligned_free(ptr);
#else
    free(ptr);
#endif
```

...so I assume we can just do:

```cpp
#if defined(ARCH_OS_WINDOWS)
    _aligned_free(ptr);
#else
    free(ptr);
#endif
```

Reply:
Unless I guess you'd rather preserve the `#if` check symmetry with above?

Reply:
I like preserving the symmetry, I think it makes it a bit more readable.